batch-nft-reveal/app/components/create/Create.tsx

```diff
@@ -0,0 +1,272 @@
+              </FormErrorMessage>
+            )}
+            <span></span>
+          </FormControl>
+          <FormControl
+            mt="4"
+            isInvalid={
+              formik.touched.vrfSubscriptionId &&
+              !!formik.errors.vrfSubscriptionId
+            }
+          >
+            <FormLabel htmlFor="vrfSubscriptionId">VRF ID</FormLabel>
+            <Tooltip
+              label="Chainlink VRF subscription ID"
+              placement="right-start"
+              fontSize="xs"
+              hasArrow
+            >
+              <div>
+                <Field
+                  as={Input}
+                  bg="white"
+                  name="vrfSubscriptionId"
+                  validate={isNotNumber}
+                />
+              </div>
+            </Tooltip>
+            {formik.errors.vrfSubscriptionId &&
+              formik.touched.vrfSubscriptionId && (
+                <FormErrorMessage>
+                  VRF ID {formik.errors.vrfSubscriptionId}
+                </FormErrorMessage>
+              )}
+          </FormControl>
+          <Divider
+            id="create-divider"
+            orientation="horizontal"
+            margin="40px 20px 20px 0px"
+            border="width 10px"
+          />
+          <Center>
+            <Tooltip
+              label="If the following conditions are met, batch reveal will be fulfilled."
+              placement="right-start"
+              fontSize="xs"
+              hasArrow
+            >
+              Batch Reveal
+            </Tooltip>
+          </Center>
+          <FormControl
+            mt="4"
+            isInvalid={
+              formik.touched.revealBatchSize && !!formik.errors.revealBatchSize
+            }
+          >
+            <FormLabel htmlFor="revealBatchSize">Batch Size</FormLabel>
+            <Field
+              as={Input}
+              bg="white"
+              name="revealBatchSize"
+              validate={isNotNumber}
+            />
+            {formik.errors.revealBatchSize &&
+              formik.touched.revealBatchSize && (
+                <FormErrorMessage>
+                  Batch Size {formik.errors.revealBatchSize}
+                </FormErrorMessage>
+              )}
+          </FormControl>
+          <FormControl
+            mt="4"
+            isInvalid={
+              formik.touched.revealInterval && !!formik.errors.revealInterval
+            }
+          >
+            <FormLabel htmlFor="revealInterval">Interval(Seconds)</FormLabel>
+            <Field
+              as={Input}
+              bg="white"
+              name="revealInterval"
+              validate={isNotNumber}
+            />
+            {formik.errors.revealInterval && formik.touched.revealInterval && (
+              <FormErrorMessage>
+                Interval(Seconds) {formik.errors.revealInterval}
+              </FormErrorMessage>
+            )}
+          </FormControl>
+          <Tooltip
```

Comment:
How it is:
<img width="192" alt="Screenshot 2022-08-24 at 17 51 27" src="https://user-images.githubusercontent.com/482351/186450392-74212f94-e41e-43f3-ae93-b34efef623d2.png">

How it should be:
<img width="147" alt="Screenshot 2022-08-24 at 17 53 30" src="https://user-images.githubusercontent.com/482351/186450868-0a79cf86-e672-4d54-98b2-7055cc6b06c5.png">